pandas/core/generic.py

```diff
@@ -6392,6 +6396,11 @@ def copy(self, deep: bool = True) -> Self:
         Series or DataFrame
             Object type matches caller.
 
+        See Also
+        --------
+        DataFrame.copy : Make a copy of this object's indices and data.
+        Series.copy : Make a copy of this object's indices and data.
```

Comment:
This is referencing the existing method. I would just suggest `copy.copy` from the standard library